### ISSUE TITLE
Heidi SQL 12.0.0.6537 keeps hanging and crashing

### ISSUE THREAD

**Issue by jlevitas16** [HeidiSQL/HeidiSQL#1648]
I upgraded a week ago to Heidi SQL 12.0.0.6537 and I can't do more than one query until the software hangs, part of the screen turns black or it crashes. I had to roll back to 12.0.0.6468 to work.

**Comment by ansgarbecker** [comment]
12.0 is not the current version. Please update to 12.1

If these issues persist, please describe exactly what you did prior to these crashes. That's what the issue template asked for, which you deleted.

**Comment by jlevitas16** [comment]
Apologies. I was using 12.1.0.6537.

I had to roll back to 12.0.0.6468.

Basically what would happen is I would open a database connection, choose a database, click on a table and run a query in a query window.

The query would work, but if I tried to run the query again, after making a change through a browser software call, it would do nothing. 

If clicked on another table, the main screen would not change to show that table. I couldn’t click within the query test area, and basically nothing would work.

I couldn’t even close HeidiSQL. I had to go to Task Manager to shut down the process.

I tried this many times and even rebooted my machine.

Also, sometimes, when I switched windows to my browser, I’d come back to Heidi and the query pane and the list of tables pane where completely black. If I clicked around, I could eventually read them again, but it didn’t matter because nothing would work. There was no actual crash – I’ve seen those plenty of times – just everything stopped working. I had no way to submit a report.

Once I rolled back to 12.0.0.6468, everything worked fine again.
